src/iexamine.cpp

```diff
@@ -3502,6 +3504,9 @@ void iexamine::autodoc( player &p, const tripoint &examp )
         popup( _( "No patient found located on the connected couches.  Operation impossible.  Exiting." ) );
         return;
     }
+    if( p.has_trait( trait_NOPAIN ) || p.has_bionic( bionic_id("bio_painkiller") ) ) {
```

Comment:
Whitespace around `"bio_painkiller"` will probably be needed too to avoid Jenkins style fail